tests/test_sanity.py

```diff
@@ -33,10 +37,21 @@ def add_to_config(content):
   with open(CONFIG_FILE, 'a') as f:
     f.write(content + '\n')
 
-
 def mtime(filename):
   return os.stat(filename).st_mtime
 
+
+@contextlib.contextmanager
+def chdir(dir):
```

Comment:
Talked in-person, probably better to not add a dependency on `mock`, but to write another wrapper for `patchEnv` or so

Reply:
Yeah, I think that will be cleaner.